src/Microsoft.ML.StandardLearners/Standard/SdcaBinary.cs

```diff
@@ -1713,18 +1727,26 @@ internal sealed class LegacySdcaBinaryTrainer : SdcaBinaryTrainerBase<IPredictor
         /// </summary>
         public sealed class Options : BinaryOptionsBase
         {
-            [Argument(ArgumentType.Multiple, HelpText = "Loss Function", ShortName = "loss", SortOrder = 50)]
-            public ISupportSdcaClassificationLossFactory LossFunction = new LogLossFactory();
+            [Argument(ArgumentType.Multiple, Name = "LossFunction", HelpText = "Loss Function", ShortName = "loss", SortOrder = 50)]
+            public ISupportSdcaClassificationLossFactory LossFunctionFactory = new LogLossFactory();
+
+            public ISupportSdcaClassificationLoss LossFunction;
 
             [Argument(ArgumentType.AtMostOnce, HelpText = "The calibrator kind to apply to the predictor. Specify null for no calibration", Visibility = ArgumentAttribute.VisibilityType.EntryPointsOnly)]
             internal ICalibratorTrainerFactory Calibrator = new PlattCalibratorTrainerFactory();
 
             [Argument(ArgumentType.AtMostOnce, HelpText = "The maximum number of examples to use when training the calibrator", Visibility = ArgumentAttribute.VisibilityType.EntryPointsOnly)]
             public int MaxCalibrationExamples = 1000000;
+
+            public Options()
```

Comment:
Please add user-facing document.

Reply:
this is removed

---
In reply to: [262724506](https://github.com/dotnet/machinelearning/pull/2856#discussion_r262724506) [](ancestors = 262724506)